inference-engine/README.md

```diff
@@ -29,31 +29,34 @@
 - [Additional Resources](#additional-resources)
 
 ## Introduction
-The Inference Engine can infer models in different formats with various input and output formats.
 
-The open source version of Inference Engine includes the following plugins:
+The Inference Engine can infer models in different formats with various input 
+and output formats.
+
+The open-source version of Inference Engine includes the following plugins:
```

Comment:
No hyphenation of "open source" even as an adjective according to https://software.intel.com/en-us/dx-hub/copy-style-guide

Reply:
Version here is used as an adjective; that means open-source is an adverb, which would need the hyphen (adverbs modifying adjectives need hyphenation).

Reply:
P.S.  I don't think it's that important to hold up the PR over though.  There are some contexts where open-source software versioning needs to be very specific.